src/components/panel/panel.js

```diff
@@ -738,29 +761,28 @@ MdPanelRef.prototype.detachOnly = function() {
 
 
 /**
- * Add a class to the panel. This can control the panel's visibility.
+ * Shows the panel.
  *
- * @param {string} newClass Class to be added.
  * @returns {!angular.$q.Promise} A promise that is resolved when the panel has
- * the new class and animations finish.
+ * shown and animations finish.
  */
-MdPanelRef.prototype.addClass = function(newClass) {
+MdPanelRef.prototype.show = function() {
```

Comment:
Internally, don't `show` and `hide` simply need to call `addClass` and do some extra stuff?
```js
MdPanelRef.prototype.show = function() {
  return this.removeClass('internal-hidden-class').then( /** show backdrop? */ );
}
```